README.rst

```diff
@@ -0,0 +1,121 @@
+|Icon| |title|_
+===============
+
+.. |title| replace:: diffpy.nmf_mapping
+.. _title: https://diffpy.github.io/diffpy.nmf_mapping
+
+.. |Icon| image:: https://avatars.githubusercontent.com/diffpy
+        :target: https://diffpy.github.io/diffpy.nmf_mapping
+        :height: 100px
+
+|PyPi| |Forge| |PythonVersion| |PR|
+
+|CI| |Codecov| |Black| |Tracking|
+
+.. |Black| image:: https://img.shields.io/badge/code_style-black-black
+        :target: https://github.com/psf/black
+
+.. |CI| image:: https://github.com/diffpy/diffpy.nmf_mapping/actions/workflows/main.yml/badge.svg
+        :target: https://github.com/diffpy/diffpy.nmf_mapping/actions/workflows/main.yml
+
+.. |Codecov| image:: https://codecov.io/gh/diffpy/diffpy.nmf_mapping/branch/main/graph/badge.svg
+        :target: https://codecov.io/gh/diffpy/diffpy.nmf_mapping
+
+.. |Forge| image:: https://img.shields.io/conda/vn/conda-forge/diffpy.nmf_mapping
+        :target: https://anaconda.org/conda-forge/diffpy.nmf_mapping
+
+.. |PR| image:: https://img.shields.io/badge/PR-Welcome-29ab47ff
+
+.. |PyPi| image:: https://img.shields.io/pypi/v/diffpy.nmf_mapping
+        :target: https://pypi.org/project/diffpy.nmf_mapping/
+
+.. |PythonVersion| image:: https://img.shields.io/pypi/pyversions/diffpy.nmf_mapping
+        :target: https://pypi.org/project/diffpy.nmf_mapping/
+
+.. |Tracking| image:: https://img.shields.io/badge/issue_tracking-github-blue
+        :target: https://github.com/diffpy/diffpy.nmf_mapping/issues
+
+Run NMF analysis on PDF and XRD data
+
+* LONGER DESCRIPTION HERE
```

Comment:
This doesn't seem to have been taken care of.

Reply:
I will address this in a separate PR.